ci/jenkins/test.sh

```diff
@@ -462,21 +462,9 @@ function deliver_antrea_linux_containerd {
 }
 
 function deliver_antrea_windows_containerd {
-    echo "===== Build Antrea Windows on Windows Jumper Node ====="
-    echo "==== Reverting Windows VM ${WIN_IMAGE_NODE} ====="
-    revert_snapshot_windows ${WIN_IMAGE_NODE}
+    echo "===== Build Antrea Windows ====="
     rm -f antrea-windows.tar.gz
-    # Compress antrea repo and copy it to a Windows node
-    mkdir -p jenkins
-    tar --exclude='./jenkins' -czf jenkins/antrea_repo.tar.gz -C "$(pwd)" .
-    for i in `seq 2`; do
-        timeout 2m scp -o StrictHostKeyChecking=no -T jenkins/antrea_repo.tar.gz Administrator@${IP}: && break
-    done
-    ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "docker pull ${DOCKER_REGISTRY}/antrea/golang:${GO_VERSION}-nanoserver && docker tag ${DOCKER_REGISTRY}/antrea/golang:${GO_VERSION}-nanoserver golang:${GO_VERSION}-nanoserver"
-    ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "rm -rf antrea && mkdir antrea && cd antrea && tar -xzf ../antrea_repo.tar.gz > /dev/null && NO_PULL=${NO_PULL}; DOCKER_NETWORK=host make build-windows && docker save -o antrea-windows.tar antrea/antrea-windows:latest && gzip -f antrea-windows.tar" || true
-    for i in `seq 2`; do
-        timeout 2m scp -o StrictHostKeyChecking=no -T Administrator@${IP}:antrea/antrea-windows.tar.gz . && break
-    done
+    make build-windows
```

Comment:
We need to export the images and copy them to remote windows worker nodes.

Reply:
make build-windows without specifying env "WINDOWS_PUSH" will save the file locally automatically.

Reply:
What I mean is that the antrea image should be copied to the Windows nodes. On line 478, I see that you deleted the `scp` command.

Reply:
No, line 478 (what I have deleted) is to copy the tar file from Windows image VN to the current VM, which was suitable in the previous solutions. And line 503 is to deliver the built antrea-agent image to every K8s Windows Node.

Reply:
> No, line 478 (what I have deleted) is to copy the tar file from Windows image VN to the current VM, which was suitable in the previous solutions. And line 503 is to deliver the built antrea-agent image to every K8s Windows Node.

My bad, but where did you create the new antrea-windows.tar.gz? I think `make build-windows` only builds the Docker image?

Reply:
> I think make build-windows only builds the Docker image?

No, make build-windows will build the image and save the tar file to path "antrea/" because I set the output=type=docker,dest=./antrea-windows.tar. Remember, this is using multi-platform buildx based on a Linux VM, docker can't use local registry to store Windows images. So the build command result is either save the image locally or push to the remote registry. In build-windows command, I make it to save files locally.  And for the push requirement, we shall use `build-window.sh --target agent --agent-tag $(DOCKER_IMG_VERSION) --push` or set environment variable "WINDOWS_PUSH=1"

Reply:
> > I think make build-windows only builds the Docker image?
> 
> No, make build-windows will build the image and save the tar file to path "antrea/" because I set the output=type=docker,dest=./antrea-windows.tar. Remember, this is using multi-platform buildx based on a Linux VM, docker can't use local registry to store Windows images. So the build command result is either save the image locally or push to the remote registry. In build-windows command, I make it to save files locally. And for the push requirement, we shall use `build-window.sh --target agent --agent-tag $(DOCKER_IMG_VERSION) --push` or set environment variable "WINDOWS_PUSH=1"

Thanks, I understand why we need a remote registry now. But I mean did you create antrea-windows.tar.gz for the next step in the CI process? From your code in build-windows.sh and build-utils.sh, I only found a .tar file.

Reply:
Got it, will update.